PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+Provide a brief description of the PR's purpose here.
+
+## Todos
+Notable points that this PR has either accomplished or will accomplish.
+* **Developer Interest**
+  - [ ] Feature1
+* **User-Facing for Release Notes**
+  - [ ] Feature2
+
+## Questions
+- [ ] Question1
+
+## Status
+- [ ] Ready to go
```

Comment:
Sure, but leave the fields flexible. I dont want it to be difficult to make a PR.